tests/rt.sh

```diff
@@ -396,11 +396,13 @@ if [[ $TESTS_FILE =~ '35d' ]]; then
 fi
 
 if [[ $MACHINE_ID = hera.* ]] || [[ $MACHINE_ID = orion.* ]] || [[ $MACHINE_ID = cheyenne.* ]] || [[ $MACHINE_ID = gaea.* ]] || [[ $MACHINE_ID = jet.* ]]; then
-  RTPWD=${RTPWD:-$DISKNM/NEMSfv3gfs/develop-20201202/${RT_COMPILER^^}}
+  RTPWD=${RTPWD:-$DISKNM/NEMSfv3gfs/develop-20201204/${RT_COMPILER^^}}
 else
-  RTPWD=${RTPWD:-$DISKNM/NEMSfv3gfs/develop-20201202}
+  RTPWD=${RTPWD:-$DISKNM/NEMSfv3gfs/develop-20201204}
 fi
 
+INPUTDATA_ROOT=${INPUTDATA_ROOT:-$DISKNM/NEMSfv3gfs/input-data-20201201/}
```

Comment:
elsewhere (utests),
`INPUTDATA_ROOT=${RTPWD}`

So, should this be:
`INPUTDATA_ROOT=${INPUTDATA_ROOT:-${RTPWD:-$DISKNM/NEMSfv3gfs/input-data-20201201/}}`

Reply:
@aerorahul, the changes are to separate input data from RTPWD,  so we don't want INPUTDATA_ROOT to be $RTPWD.